### DISCUSSION TITLE
Meshes in Devito

### DISCUSSION THREAD

**Discussion by GSPaulo** [devitocodes/devito#2360] [General]
Dear all,
I am starting using Devito and I would like to know what meshes I can use in Devito and if I can import meshes from Gmesh, for example.
Thank you a lot,
GSPaulo
Gilcilene Sanchez de Paulo

**Comment by mloubout** [comment]
Hello

At the time devito supports structure Cartesian grids. Unstructured mesh are on the roadmap but not in the near future projects as it will require major changes.

**Reply by GSPaulo** [reply]
Hello Mathias Louboutin. Thanks for your attention.

I tried to engage with the Devito user/dev community on [Slack](https://join.slack.com/t/devitocodes/shared_invite/zt-gtd2yxj9-Y31YKk_7lr9AwfXeL2iMFg), but this link is not working anymore.

I am going to start using Devito, so I think I will have some doubts. Have you another link or Could I continue asking here (for me no problem - it is ok here)?

I have one more question, but sincerely I really need to watch the video https://youtu.be/druSsV_1O6w 
in details, maybe on this video I can find the answer.

I will ask here now, but do not worry to answer it if the topic has been discussed on the video.
1) What is/are the possibility/ies of  arrangement of the variables on the mesh (co-located or staggered)? 

Thank you so much for your attention, 
Gilcilene Sanchez de Paulo (Unesp/Brazil)

**Reply by mloubout** [reply]
Hi

We support both Cartesian and staggered grid and combination of. 

The slack link was indeed outdated we made a new one last week that should work now
https://join.slack.com/t/devitocodes/shared_invite/zt-2hgp6891e-jQDcepOWPQwxL5JJegYKSA

Cheers

**Reply by GSPaulo** [reply]
Hello! The slack link worked, I am there now! 
Good luck to my group in this new experience with Devito!
Thanks for all.
